### PR TITLE
Updating to package template 1.1.2

### PR DESCRIPTION
**Pull request by bsipocz**
Includes configparser changes at both places for python 3.5.2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,27 +45,29 @@ matrix:
...
+        - python: 2.7
+          env: NUMPY_VERSION=1.9
         - python: 2.7
-          env: NUMPY_VERSION=1.8 SETUP_CMD='test --remote-data'
+          env: NUMPY_VERSION=1.8
         - python: 2.7
-          env: NUMPY_VERSION=1.7 SETUP_CMD='test --remote-data'
+          env: NUMPY_VERSION=1.7
         - python: 2.7
-          env: NUMPY_VERSION=1.6 SETUP_CMD='test --remote-data'
```
Comment (astrofrog) [inline .travis.yml]: Can you revert the ``.travis.yml`` file to what it was before? We need all the remote tests because most of the image tests use data from data.astropy.org

File: docs/conf.py [hunk 2]
```diff
@@ -91,22 +95,28 @@
 # variables set in the global configuration. The variables set in the
 # global configuration are listed below, commented out.
 
+html_theme_options = {
+    'logotext1': 'wcs',  # white,  semi-bold
+    'logotext2': 'axes',  # orange, light
+    'logotext3': ':docs'   # white,  light
+    }
```
Comment (astrofrog) [inline docs/conf.py]: It looks like this is causing a Sphinx test failure?

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [inline .travis.yml]: There is a global ``SETUP_CMD='test --remote-data'``, so I suppose the builds should be the same as before?

Comment (astrofrog) [inline .travis.yml]: Ah sorry I missed that :)

File: docs/conf.py (hunk 2 above)
Comment (bsipocz) [inline docs/conf.py]: Yes, thanks there was some rtd theme leftover.

**Comment by bsipocz** [comment]
I still have two sphinx warnings locally, and don't really know what to do with them:
```
WARNING: FITSFixedWarning: LONPOLE2= 180.000000000 /lonpole 
invalid alternate code, keyword resembles LONPOLEa but isn't. [astropy.wcs.wcs]
WARNING: FITSFixedWarning: LATPOLE2= 0.00000000000 /latpole 
invalid alternate code, keyword resembles LATPOLEa but isn't. [astropy.wcs.wcs]
```

**Comment by astrofrog** [comment]
@bsipocz - I think we just need to fix the data in the data.astropy.org repo (however I'm not sure if those warnings ever caused sphinx to fail before?)

**Comment by bsipocz** [comment]
Weird, they don't fail travis :)

However numpy 1.6 does, as we build it with the stable astropy. We can use astropy 1.1 in that build if you want to keep it, or can remove it?

**Comment by astrofrog** [comment]
@bsipocz - thanks! One small request - since 1.1.1post isn't actually a package-template release, can you rephrase the commit message to just say 'to the latest developer version of package-template'? Alternatively, I think it would be best to actually tag 1.1.2 in the package-template. If I do that, can you update the commit message to then say 1.1.2?

**Comment by bsipocz** [comment]
Calling it simply the latest dev version is not very helpful I think, but tagging the template before mass applying sounds like a good plan.  The configparser changes are critical enough to justify 1.1.2 anyway :)

**Comment by astrofrog** [comment]
@bsipocz - ok, I'll tag it later today (tags in the package-template are easy enough to do, unlike a core package release!). I just need to update the changelog first.

**Comment by astrofrog** [comment]
@bsipocz - I tagged v1.1.2, I'll announce it on the affiliated package maintainer mailing list later.

**Comment by astrofrog** [comment]
@bsipocz - thanks!
